test/__snapshots__/modules-option.test.js.snap

```diff
@@ -6355,6 +6355,33 @@ exports.locals = {
 
 exports[`modules issue #861: warnings 1`] = `Array []`;
 
+exports[`modules issue #966: errors 1`] = `Array []`;
+
+exports[`modules issue #966: module (evaluated) 1`] = `
+Array [
+  Array [
+    1,
+    ".button.hey {
```

Comment:
the bug is right here: `.button.hey` is very different than: `.button\.hey`. The former is a selector for when both `.button` and `.hey` are on an element